lib/opentelemetry-bridge/OTelTracer.js

```diff
@@ -74,7 +74,7 @@ class OTelTracer {
       }
       newTransOrSpan = trans.createSpan(name, spanOpts)
 
-      // There might be no span, e.g. if transactionMaxSpans was hit. We have
+      // There might be no span, e.g. if the span is an exit span. We have
```

Comment:
Note: transactionMaxSpans being reached now creates a span -- but sets its recorded value to false.